common/src/main/java/org/keycloak/common/crypto/CryptoIntegration.java

```diff
@@ -54,7 +55,7 @@ private static CryptoProvider detectProvider(ClassLoader classLoader) {
             throw new IllegalStateException("Multiple crypto providers loaded with the classLoader: " + classLoader +
                     ". Make sure only one cryptoProvider available on the classpath. Available providers: " +foundProviders);
         } else {
-            logger.infof("Detected crypto provider: %s", foundProviders.get(0).getClass().getName());
+            logger.debugf("Detected crypto provider: %s", foundProviders.get(0).getClass().getName());
```

Comment:
I had to set this to debug because the KcRegTest and KcAdmTest both failed as they are expecting only a single line in the response. If the log is at info it adds an additional line to the response and causes assertion failures in several tests.

Reply:
I wonder how is it possible? As before this PR, there was INFO log and the tests were still passing. And AFAIK the base testsuite still uses `crypto/default` by default. In theory, addition of `crypto/elytron` module should not change anything in here, or is it? So I wonder why is it affected by these changes?

I guess it can be eventually possible to change the tests then to expect 2 lines instead of 1, but it will be good to know why the `KcRegTest` and `KcAdmTest` were affected by changes in this PR...

Reply:
I'm sure it had to do with the changes I made to the PemUtils and/or DerUtils to use CryptoIntegration.

Reply:
Ah, ok. Thanks. So is it then possible to change the tests to expect 2 lines instead of 1?